static/src/javascripts/projects/commercial/modules/dfp/dfp-env.js

```diff
@@ -63,4 +63,4 @@ const dfpEnv: DfpEnv = {
     },
 };
 
-export default dfpEnv;
+export { dfpEnv };
```

Comment:
If we do that then presumably we can get rid of the type DfpEnv as well and just type each individual element of the object?